src/rp2_common/pico_standard_link/binary_info.c

```diff
@@ -66,7 +66,13 @@ bi_decl(bi_program_url(PICO_PROGRAM_URL))
 #endif
 #endif
 
-#if !PICO_NO_BUILD_TYPE_FEATURE
```

Comment:
Had to go looking to see where PICO_NO_BUILD_TYPE_FEATURE existed, seems this was a historic typo? Nice

This does raise that there are no PICO_CONFIG metacomments for the bininfo defines. Want me to add those?

Reply:
sure